docs/reference/module-api/module-overview.md

```diff
@@ -103,21 +94,84 @@ Set to `false` to prevent the module from being fully instantiated in the applic
+
+``` bash
+// in your project, already npm installed
+
+cd node_modules/vue-material-design-icons
+ls *.vue
+```
+Each property key in the `icons` setting object will be the name used to reference the icon in an Apostrophe project. The value will be the Material Design name for the icon, written in PascalCase without the `.vue` ending. The Apostrophe reference name for the icon *does not need to match* the Material Design name.
+
+```javascript
+// index.js
+module.exports = {
+  // ...
+  icons: {
+    airhorn: 'AirHorn',
+    expander: 'ArrowUpDownBoldOutline'
+  }
+};
+```
+
+To use an icon that is not included in the `vue-material-design-icons` list, add your icon Vue file to either a relative path in the project or via an `npm` package. Then register the icon with a property name that will be used to reference the icon in the project, and a value that points to the file.
+
+```javascript
+icons: {
+  // For an icon at ./icons in your project
+  'my-icon': '~./icons/MyIconName.vue',
+  // For an icon in the their-icon-bundle-package npm module
+  'their-icon': '~their-icon-bundle-package/TheirIconName.vue'
+}
+```
+Everything following the `~` becomes part of an `import` statement during the build process.
+
+If you need to convert your icon(s) to Vue components, you can use any of the icons in the `vue-material-design-icons` as a template for constructing a simple wrapper.
+
+At the present time, the same icon cannot be registered under two names (that is, it can't be registered as `my-icon` and as `core-icon` if they both refer to the same icon). Since this can be inconvenient and requires checking the `globalicons.js` file to make sure you are not registering a duplicate, we plan to correct it in an upcoming release.
+
+## Configuration cascades
+
+| Setting name | Value type | Description | Module types |
+| ------- | ------- | ------- | ------- | :-----: |
+| [`fields`](#fields) | Object/Function | Configure doc type fields | Doc, Widget |
+| [`filters`](#filters) | Object/Function | Configure piece type filters | Piece |
+| [`columns`](#columns) | Object/Function | Configure piece type manager columns | Piece |
+| [`batchOperations`](#batchoperations) | Object/Function | Configure manager batch operations | Piece |
+
+### "Cascading" settings
+These settings can either be configured as a static object or through a function that takes `self` and `options` and returns a configuration object.
+
+As detailed for each setting, the configuration objects have `add`, `remove`, `group`, and `order` properties. This pattern allows these settings to "cascade" from the base classes to project-level classes without requiring those settings be declared again.
+
+Use `add` to add additional settings and `remove` to remove existing base class settings. Use `group` to organize user-facing settings in the editing interface. The `order` option only applies to columns, arranging columns in a particular order.
```

Comment:
`order` is useful with any cascade that doesn't need `groups`, which would also include `filters` and `batchOperations`.

Reply:
Changed/